src/status.js

```diff
@@ -20,6 +18,7 @@ var debounce = function (func, wait, immediate) {
 
 export default class Status extends Component {
   static defaultProps = {
+    statusDebounceMillis: 1400,
```

Comment:
Is this definitely something we think needs to be configurable? Is that something that we actually expect users to change, or just for testing purposes? Generally, it'd be good to try and avoid exposing props just for testing purposes as they then become part of the public API.

If we do think there's a good reason to keep this around, can we make sure it's added to the API docs?

Reply:
Sure - will remove this from props.

Reply:
Now removed from props.